### PR TITLE
Configuration

### PR DESCRIPTION
**Pull request by jayastronomic**

### CODE REVIEW

**Review by jayastronomic — COMMENTED** [review]
Everything looks good!

**Review by kairoje — APPROVED** [review]
Make updates to erd, and readme.md

File: ERD.png
Comment (lizabawa) [inline ERD.png:1]: ERD diagram needs to be updated

**Review by lizabawa — CHANGES_REQUESTED** [review]
Readme needs to be refactored from "items" to "expenses", and ERD diagram needs to be updated to the latest version

**Review by lizabawa — APPROVED** [review]

**Pull request merged**